src/crypto/x509.js

```diff
@@ -43,6 +43,10 @@ const keyPurposes = {
     emailProtection: "1.3.6.1.5.5.7.3.4",
 }
 
+const OIDs = {
```

Comment:
Let's add more meaningful name to this constant.
According to the OID repository parent value (2.5.4) called attribute type
http://oid-info.com/get/2.5.4

I suggest to rename this constant to:
const attributeTypes = { ... }